src/demo/SdkDemo.tsx

```diff
@@ -46,10 +57,27 @@ const SdkDemo: FunctionComponent<Props> = ({
     const url = getTokenFactoryUrl(builtRegionCode)
     setTokenUrl(url)
 
-    getToken(hasPreview, url, messagePort, (respondedToken) =>
-      setToken(respondedToken)
+    getToken(
```

Comment:
btw we should make sure that this function is only called if this is not the `mobileFlow` because we use this same demo app to test local cross device flow

Reply:
Yeah I also got the same situation when testing with cross-device flow locally. But eventually, this doesn't affect the flow at all because SDK Token will be replaced with the one sent from desktop.